src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.css

```diff
@@ -32,3 +32,7 @@
   width: 50px;
   margin-right: 10px;
 }
+
+.quickMarcFocusedRow {
+  background: rgba(114, 150, 226, 0.3);
```

Comment:
Is there a specific requirement for this color? There is a similar named var `--color-fill-focus: rgba(37, 118, 195, 0.3);`